spec/controllers/entities_controller_spec.rb

```diff
@@ -490,8 +491,17 @@
       login_user
 
       before do
-        @entity = double('entity', :name => 'my awesome name')
-        expect(Entity).to receive(:find).and_return(@entity)
+        # NOTE: @aguestuser twists himself into contortions to write
+        # these mocks to fix controller specs that fail for functionality that works
+        # *and* has passing feature specs and wonders...
+        # ... why are we writing tests that are as brittle as this anyway?
+        # ... why are we writing tests that break when an implementation changes but the result does not?
+        # ... do they *really* give us a lot of value we could not get from a less brittle approach?
+        @entity = double('entity', :name        => 'my awesome name',
```

Comment:
> NOTE: @aguestuser twists himself into contortions to write
> these mocks to fix controller specs that fail for functionality that works
>  *and* has passing feature specs and wonders...
> ... why are we writing tests that are as brittle as this anyway?

I agree that we don't need controller tests if we have full feature specs. I wrote a lot of these controller tests before I understood and used the philosophy of writing only feature specs.  Use the converge report tool to see if the controller tests are testing code that isn't covered by a feature spec. If we have a feature spec for it then you can delete the controller tests, otherwise your options are to either adapt the controller tests or write new features specs.

> ... why are we writing tests that break when an implementation changes but the result does not?
>  ... do they *really* give us a lot of value we could not get from a less brittle approach?

See above comment. I sometimes find controller tests to still be useful for testing certain tricky controller methods, but I agree that these particular ones aren't that helpful. 

A small meta-meta-meta comment 🌵:  these comments buried in the middle of the spec file are probablyn not the best place for this conversion...they are totally valid observations about the specs, but will likely but lost there. perhaps we should start a page on the wiki or another github issue with strategies on how to avoid writing brittle tests.

Reply:
valid meta-meta point. wanted to write the observation where it occured to me and where i thought you'd notice them in a context that will make sense. when refactoring for merge, i'll delete them.